docs/hsi.md

```diff
@@ -372,21 +372,39 @@ To meet HSI-1 on systems that run this test, the result must be `not-enabled`. *
 - [Chipsec 4xxlp register definitions](https://github.com/chipsec/chipsec/blob/master/chipsec/cfg/8086/pch_4xxlp.xml#L270)
 - [RISC-V EDK PCH register definitions](https://github.com/riscv/riscv-edk2-platforms/blob/85a50de1b459d1d6644a402081120770aa6dd8c7/Silicon/Intel/CoffeelakeSiliconPkg/Pch/Include/Register/PchRegsDci.h)
 
+<a id="org.fwupd.hsi.Fused"></a>
+
+### [Part is fused](#org.fwupd.hsi.Fused)
```

Comment:
I'm wondering if this should be org.fwupd.hsi.CpuFused instead

Reply:
It's not a property of individual CPU cores or even chiplets. How about FusedSoc? Or Fused Part?

Reply:
What about org.fwupd.hsi.PlatformFused?

Reply:
That's good, will swap.

Also what do you think about me splitting this PR into 2 - all the new attributes and the new plug-in?

I just thought it made sense to see them in context is why it's all in one.

Reply:
Yup, definitely a split; leave the new plugin here and split out the other stuff we want in tree ASAP.